src/main/resources/db/migration/V16__20251128__create_mentee_mentorship_focus.sql

```diff
@@ -0,0 +1,8 @@
+
+-- TABLE Mentee mentorship focus areas (join table)
+CREATE TABLE mentee_mentorship_focus_areas
```

Comment:
Because you didn't merge yet, you could move this change to your migration 15; Locally you can just remove your database and you won't have any problem ;)